src/settings/FundTypeSettings/FundTypeSettings.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+  injectIntl,
+  intlShape,
+} from 'react-intl';
+import { Field } from 'redux-form';
+
+import { ControlledVocab } from '@folio/stripes/smart-components';
+import {
+  TextArea,
+  TextField,
+} from '@folio/stripes/components';
+
+import { FUND_TYPES_API } from '../../common/const';
+import { fundTypesResource } from '../../common/resources';
+
+const fieldComponents = {
+  // eslint-disable-next-line react/prop-types
+  name: ({ fieldProps }) => {
+    return (
+      <Field
+        {...fieldProps}
```

Comment:
it's like you always overriding props, usually it's a potential bug

Reply:
no overriding here, just extending